Sources/FluentMySQLDriver/FluentMySQLConfiguration.swift

```diff
@@ -5,7 +5,7 @@ extension DatabaseConfigurationFactory {
     public static func mysql(
         unixDomainSocketPath: String,
         username: String,
-        password: String,
+        password: String? = nil,
```

Comment:
```suggestion
        password: String?,
```

Do not default the password to nil - we don't want to encourage use of empty passwords.